### PR TITLE
x86_cpu_flags:test SPR new flags

### PR DESCRIPTION
**Pull request by nanliu-r**
ID: 1965163

Signed-off-by: Nana Liu <nanliu@redhat.com>

### CODE REVIEW

**Comment by nanliu-r** [comment]
@vivianQizhu @huangyum Would you please help to review the code? the machine is being updated and I can't pass all flags now because of the product bugs and hardware issue. thanks a lot.

File: qemu/tests/cfg/x86_cpu_flags.cfg [hunk 1]
```diff
@@ -60,6 +60,15 @@
                                 cpu_model_flags += ",intel_pt=on"
                             i440fx:
                                 cpu_model_flags += ",intel_pt=on,min-level=0x14"
+                - test_SPR_flags:
+                    required_qemu = [5.2.0, )
+                    flags = "serialize avx512_vp2intersect avx512_fp16 tsxldtr"
```
Comment (huangyum) [inline qemu/tests/cfg/x86_cpu_flags.cfg:65]: Would you pls double check 'tsxldtr'? I don't see you test this flag in the rfe bug.

Comment (nanliu-r) [reply qemu/tests/cfg/x86_cpu_flags.cfg:65]: > Would you pls double check 'tsxldtr'? I don't see you test this flag in the rfe bug.

As we discuss in bz, the 'tsxdtr' will can be checked on host(need to be upgrade) and I can check it in code now.

**Review by huangyum — APPROVED** [review]
ACK

**Comment by vivianQizhu** [comment]
Thanks @nanliu-r and @huangyum

**Pull request merged**